client/my-sites/domains/email-providers-upsell/index.jsx

```diff
@@ -10,47 +10,53 @@ import { useTranslate } from 'i18n-calypso';
  * Internal dependencies
  */
 import { domainAddNew } from 'calypso/my-sites/domains/paths';
+import CalypsoShoppingCartProvider from 'calypso/my-sites/checkout/calypso-shopping-cart-provider';
```

Comment:
Nit: please add this before `domainAddNew`.
```suggestion
import CalypsoShoppingCartProvider from 'calypso/my-sites/checkout/calypso-shopping-cart-provider';
import { domainAddNew } from 'calypso/my-sites/domains/paths';
```